### DISCUSSION TITLE
jgmenu in qtile?

### DISCUSSION THREAD

**Discussion by ervinpopescu** [qtile/qtile#3699] [Q&A] [answered]
Does `qtile` have the concept of a root window like `dwm` and other WMs?

I saw plenty of rices over on `r/unixporn` which featured `jgmenu`, an application launcher (and much more) which can be called when clicking on the "background", i.e. the root window, and was wondering if that could work in `qtile`.

**Comment by m-col** [comment, marked as answer]
Yep that should work. It should work in any standalone X11 WM, I suspect what can interfere with that functionality is background windows, which are the ones that DEs use to draw icons and widgets on, for example.

**Reply by ervinpopescu** [reply]
Well, OOTB it doesn't. Do you know if it requires additional configuration?

**Reply by m-col** [reply]
What are you using to call jgmenu upon click of the root window?

**Reply by m-col** [reply]
You will need to trigger it with something like https://github.com/phillbush/xclickroot

**Reply by ervinpopescu** [reply]
> You will need to trigger it with something like https://github.com/phillbush/xclickroot

Thanks, that's what I was looking for :)

**Reply by ervinpopescu** [reply]
One more question @m-col, can I run this in my autostart script or do I have to follow the instructions and run it in .xsession/.xinit?

LE: it seems to work fine in `autostart`.

**Reply by m-col** [reply]
For posterity - if you're forking it off into the backgroudn you can run it anywhere where the right `$DISPLAY` variable is set
